### DISCUSSION TITLE
Can Lantern run at Harmony OS

### DISCUSSION THREAD

**Discussion by ttcath** [getlantern/lantern-forum-cn#951] [General]
Can Lantern run at Harmony OS Next? If could, pls provide the link or method. thx

**Comment by MaxMKMax** [comment]
Hi @ttcath , i will help to check with the technical team and i will get back to you with some useful guides and instructions to run it. Thanks for your inquiry and waiting. 

Have a great day! Thank you.

**Reply by ttcath** [reply]
Dear @Mike777Khoo  Thank you so much. Please note: in Harmony OS Next, they cannot install any .apk file in it.
